### PR TITLE
add design

### PR DESCRIPTION
**Pull request by v-nakon**

### CODE REVIEW

File: src/style.css [hunk 1]
```diff
@@ -116,4 +116,118 @@
 
 .visibility-icon-menu {
   visibility: hidden;
+}
+
+.container-slider {
+  padding-top: 60px;
```
Comment (byzyk) [inline src/style.css]: better to use `margin` here

File: src/style.css [hunk 2]
```diff
@@ -116,4 +116,118 @@
...
+  font-style: normal;
+  font-weight: bold;
+  line-height: 70px;
+  font-size: 60px;
+  letter-spacing: -1px;
+  color: #151515;
+  padding-top: 50px;
+}
+
+.text1 {
```
Comment (byzyk) [inline src/style.css]: why `text1` ?

File: src/style.css [hunk 3]
```diff
@@ -116,4 +116,118 @@
...
+  width: 474px;
+  height: 81px;
+  font-family: Fira Sans;
+  font-style: normal;
```
Comment (byzyk) [inline src/style.css]: this is redundant

File: src/style.css [hunk 4]
```diff
@@ -116,4 +116,118 @@
...
+  font-weight: normal;
```
Comment (byzyk) [inline src/style.css]: same as above

File: src/style.css [hunk 5]
```diff
@@ -116,4 +116,118 @@
...
+  display: flex;
+  justify-content: center;
+  align-self: center;
+  margin-top: 56px;
+  cursor: pointer;
+}
+
+.text-button {
+  font-family: Fira Sans;
+  font-style: normal;
```
Comment (byzyk) [inline src/style.css]: same as above

File: src/style.css [hunk 6]
```diff
@@ -116,4 +116,118 @@
...
+  background: #4737FF;
+  display: flex;
+  justify-content: center;
+  align-self: center;
+  margin-top: 56px;
+  cursor: pointer;
+}
+
+.text-button {
+  font-family: Fira Sans;
```
Comment (byzyk) [inline src/style.css]: no font fallback

File: src/style.css [hunk 7]
```diff
@@ -116,4 +116,118 @@
...
+  height: 18px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 190px;
+}
+
+.loader-text {
+  font-family: Fira Sans;
+  font-style: normal;
```
Comment (byzyk) [inline src/style.css]: same

File: src/style.css [hunk 8]
```diff
@@ -116,4 +116,118 @@
...
+  font-weight: normal;
```
Comment (byzyk) [inline src/style.css]: same

File: src/style.css [hunk 9]
```diff
@@ -116,4 +116,118 @@
...
+  width: 65px;
+  height: 65px;
+  background: #FFFFFF;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.play {
+  margin-left: 3px;
```
Comment (byzyk) [inline src/style.css]: why do we need this?

Comment (v-nakon) [reply src/style.css]: element was not in center

File: src/style.css (hunk 2 above)
Comment (v-nakon) [reply src/style.css]: because we have several texts in template.
is it bad for variable names?

Comment (byzyk) [reply src/style.css]: yes, it might be confusing, it's better to use something like: `slider-text`, `text-button` etc.

File: src/style.css (hunk 9 above)
Comment (byzyk) [reply src/style.css]: Try to debug and find the reason why it wasn't centered 🙂

**Pull request merged**